pkgs/tools/security/age-plugin-yubikey/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, pcsclite
+, PCSC
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "age-plugin-yubikey";
+  version = "unstable-2021-12-24";
```

Comment:
Maybe just use `v0.2.0` from https://github.com/str4d/age-plugin-yubikey/commit/51910edfab4006a068864602469ff7db3766bfbe?

Reply:
Unfortunately there is no release or tag `v0.2.0` 😅 
I'll create a issue on their side

Reply:
Sure, but just use the commit as a `rev` and change `version` to `0.2.0`. Maybe add a comment that there's no tag for this release.